tailwind.config.js

```diff
@@ -9,5 +9,5 @@ module.exports = {
   theme: {
     extend: {},
   },
-  plugins: [],
+  plugins: [require("tailwindcss-animated")],
```

Comment:
your code is fine, just assign yourself to the PR

Reply:
Everything should be good